### PR TITLE
WIP: helm-charts/seldon-core-operator: Allow to customize controller resources

### PR DESCRIPTION
**Pull request by desaintmartin**
Also complete Chart.yaml

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **axsaucedo**

If they are not already assigned, you can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Hi @desaintmartin. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) or [seldonio](https://github.com/orgs/seldonio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/seldonio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by axsaucedo** [comment]
@desaintmartin there seem to be merge conflicts

**Comment by cliveseldon** [comment]
The helm templates are generated from the Kustomize. Using https://github.com/SeldonIO/seldon-core/blob/master/operator/helm/split_resources.py
This file would need changing.

File: helm-charts/seldon-core-operator/Chart.yaml [hunk 1]
```diff
@@ -7,3 +7,10 @@ name: seldon-core-operator
 sources:
 - https://github.com/SeldonIO/seldon-core
 version: 0.5.0-SNAPSHOT
+appVersion: 0.5.0-SNAPSHOT
```
Comment (gsunner) [inline helm-charts/seldon-core-operator/Chart.yaml]: If you can remove "appVersion". This is an optional helm item that we dont use.

**Comment by axsaucedo** [comment]
@desaintmartin would you be able to address it in the file that clive pointed?

**Comment by cliveseldon** [comment]
Superceded by #1407  so closing this

**Comment by seldondev** [comment]
failed to trigger Pull Request pipeline
* failed to create agent
* failed to calculate in repo config
* failed to load trigger config for repository SeldonIO/seldon-core for ref 2a83e485ac8a43001db84a69ce3a26864e638170
* failed to find any lighthouse configuration files in repo SeldonIO/seldon-core at sha 2a83e485ac8a43001db84a69ce3a26864e638170
* failed to process repo SeldonIO/seldon-core refref 2a83e485ac8a43001db84a69ce3a26864e638170
* failed to list files in directory /var/tmp/gitrepo283418320/.lighthouse
* open /var/tmp/gitrepo283418320/.lighthouse
* no such file or directory

**Pull request closed without merging**